### PR TITLE
orderByNulls is now GA

### LINKED ISSUE #4853: Prisma Client Docs: make `orderByNulls` GA
Part of https://github.com/prisma/prisma/issues/19377

Preview feature mentioned in
- https://github.com/prisma/docs/blob/main/content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx?plain=1
- https://github.com/prisma/docs/blob/main/content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx?plain=1

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/docs/issues/4853

### CODE REVIEW

**Review by ruheni — APPROVED** [review]

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 1]
```diff
@@ -377,20 +377,13 @@ const getUsersByRelevance = await prisma.user.findMany({
 
 <Admonition type="info">
 
+This feature is generally available in version `4.16.0` and later. It was first added as a [Preview feature](/concepts/components/preview-features) in version [`4.1.0`](https://github.com/prisma/prisma/releases/tag/4.1.0).
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]: Missing the preview feature needed to enable the feature.

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 2]
```diff
@@ -377,20 +377,13 @@ const getUsersByRelevance = await prisma.user.findMany({
...
+
 Note: Prisma Client does not support this feature for MongoDB.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx:386]: I think we usually split the version info and feature limitations in different admonitions.

**Review by janpio — CHANGES_REQUESTED** [review]
See existing comments.

File: content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx [hunk 3]
```diff
@@ -377,20 +377,19 @@ const getUsersByRelevance = await prisma.user.findMany({
 
 <Admonition type="info">
 
-Note: Prisma Client does not support this feature for MongoDB.
+This feature is generally available in version `4.16.0` and later.
+
+To use this feature in versions [`4.1.0`](https://github.com/prisma/prisma/releases/tag/4.1.0) to [`4.15.0`](https://github.com/prisma/prisma/releases/tag/4.15.0) the [Preview feature](/concepts/components/preview-features/client-preview-features#enabling-a-prisma-client-preview-feature) `orderByNulls` will need to be enabled.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/050-filtering-and-sorting.mdx]:
```suggestion
This feature is generally available in version `4.16.0` and later. To use this feature in versions [`4.1.0`](https://github.com/prisma/prisma/releases/tag/4.1.0) to [`4.15.0`](https://github.com/prisma/prisma/releases/tag/4.15.0) the [Preview feature](/concepts/components/preview-features/client-preview-features#enabling-a-prisma-client-preview-feature) `orderByNulls` will need to be enabled.
```

**Review by janpio — APPROVED** [review]

**Pull request merged**